bigquery/cloud-client/natality_tutorial.py

```diff
@@ -38,52 +38,43 @@ def run_natality_tutorial():
     # defaults.
     client = bigquery.Client()
 
-    # The name for the new dataset.
-    dataset_id = 'natality_regression'
-
-    # Prepare a reference to the new dataset.
-    dataset_ref = client.dataset(dataset_id)
+    # Prepare a reference to a new dataset for storing the query results.
+    dataset_ref = client.dataset('natality_regression')
     dataset = bigquery.Dataset(dataset_ref)
 
     # Create the new BigQuery dataset.
     dataset = client.create_dataset(dataset)
 
-    # In the new BigQuery dataset, create a new table.
+    # In the new BigQuery dataset, create a reference to a new table for
+    # storing the query results.
     table_ref = dataset.table('regression_input')
-    # The table needs a schema before it can be created and accept data.
-    # Create an ordered list of the columns using SchemaField objects.
-    schema = [
-        bigquery.SchemaField('weight_pounds', 'float'),
-        bigquery.SchemaField('mother_age', 'integer'),
-        bigquery.SchemaField('father_age', 'integer'),
-        bigquery.SchemaField('gestation_weeks', 'integer'),
-        bigquery.SchemaField('weight_gain_pounds', 'integer'),
-        bigquery.SchemaField('apgar_5min', 'integer'),
-    ]
-
-    # Assign the schema to the table and create the table in BigQuery.
-    table = bigquery.Table(table_ref, schema=schema)
-    table = client.create_table(table)
+
+    # Configure the query job.
+    job_config = bigquery.QueryJobConfig()
+
+    # Set the destination table to the table reference created above.
+    job_config.destination = table_ref
+
+    # BigQuery can auto-detect the schema based on the source table.
+    job_config.autodetect = True
```

Comment:
QueryJobConfig doesn't have an `autodetect` parameter. This line is not needed.

Sorry if my previous comment was confusing. I mean that when BigQuery runs a query, it knows the types of the various columns in the query results based on the query text, so no explicit schema is needed.

Reply:
Oops removed the nonsense attribute